OpenRA.Mods.Common/Traits/BotModules/BaseBuilderBotModule.cs

```diff
@@ -477,14 +477,16 @@ bool IsRallyPointValid(CPos producerLocation, CPos rallyPointLocation, Locomotor
 		// Require at least one refinery, unless we can't build it.
 		public bool HasAdequateRefineryCount() =>
 			Info.RefineryTypes.Count == 0 ||
-			AIUtils.CountActorByCommonName(RefineryBuildings) >= MinimumRefineryCount() ||
+			AIUtils.CountActorByCommonName(RefineryBuildings) >= OptimalRefineryCount() ||
 			AIUtils.CountActorByCommonName(powerBuildings) == 0 ||
 			AIUtils.CountActorByCommonName(ConstructionYardBuildings) == 0;
 
-		int MinimumRefineryCount() =>
+		int OptimalRefineryCount() =>
 			AIUtils.CountActorByCommonName(ProductionBuildings) > 0
 			? Info.InititalMinimumRefineryCount + Info.AdditionalMinimumRefineryCount
 			: Info.InititalMinimumRefineryCount;
+		bool MinimalRefineryCount() =>
```

Comment:
```suggestion
		bool HasMinimalRefineryCount() =>
```

To be semantically correct